src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPoolManager.cs

```diff
@@ -36,15 +36,15 @@ internal sealed class HttpConnectionPoolManager : IDisposable
         /// <summary>The pools, indexed by endpoint.</summary>
         private readonly ConcurrentDictionary<HttpConnectionKey, HttpConnectionPool> _pools;
         /// <summary>Timer used to initiate cleaning of the pools.</summary>
-        private readonly Timer _cleaningTimer;
+        private readonly Timer? _cleaningTimer;
```

Comment:
Why is it nullable if it is initialized in constructor?

Reply:
It is set conditionally in constructor